fabric-resource-loader-v0/src/main/resources/fabric.mod.json

```diff
@@ -28,6 +28,9 @@
   ],
   "accessWidener": "fabric-resource-loader-v0.accesswidener",
   "custom": {
-    "fabric-api:module-lifecycle": "stable"
+    "fabric-api:module-lifecycle": "stable",
+    "loom:injected_interfaces": {
+      "net/minecraft/class_3288": ["net/fabricmc/fabric/impl/resource/loader/FabricResourcePackProfile"]
```

Comment:
This will make an impl package interface appear to modders on a vanilla type, which encourages the use of impl methods.

Reply:
What's the best way to solve this? Mark it as ApiStatus.Internal? Or just not use II at all?